presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1841,6 +1841,16 @@ public void testCreateAndInsert()
         assertEqualsIgnoreOrder(actualAfterTransaction, expected);
     }
 
+    @Test
+    public void testAddColumn()
+    {
+        assertUpdate("CREATE TABLE test_add_column (a bigint COMMENT 'aaa')");
```

Comment:
Please replace `aaa` with something more meaningful like `test comment for column a`